main.go

```diff
@@ -211,16 +253,16 @@ func loadAPI() *openapi3.T {
 		os.MkdirAll(cacheDir, 0700)
 	}
 
-	spec := cacheDir + "cache-" + fmt.Sprintf("%x", md5.Sum([]byte(ScalrHostname))) + "-openapi-preview.yml"
+	spec := cacheDir + "cache-openapi-preview.yml"
 
 	if info, err := os.Stat(spec); !os.IsNotExist(err) {
 		if time.Since(info.ModTime()).Hours() > 24 {
 			//Cache is more than 24 hours old, re-Download...
-			downloadFile("https://"+ScalrHostname+"/api/iacp/v3/openapi-preview.yml", spec)
+			downloadFile("https://scalr.io/api/iacp/v3/openapi-preview.yml", spec)
 		}
 	} else {
 		//Download spec
-		downloadFile("https://"+ScalrHostname+"/api/iacp/v3/openapi-preview.yml", spec)
+		downloadFile("https://scalr.io/api/iacp/v3/openapi-preview.yml", spec)

```

Comment:
@buzzy can you add scalr.io as default hostname and allow it to override from the config file? Otherwise, we'll not be able to test API changes in the dev branches.